app/models/letter.rb

```diff
@@ -67,6 +64,8 @@ def format_address(address)
     end
   end
 
+  private
+
```

Comment:
Not part of your PR, but it looks like this format_letter_address method is not used in the app - I wonder if it's worth removing it, since you're touching this file anyway?